en/appbuilder/generate-app/run-application-locally.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Indigo Design App Builder - Upload Application to GitHub
+_description: Indigo Design App Builder enables users to choose between downloading their application locally or uploading it to their GitHub repository.
+_keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch UI kits, GitHub
+---
+# Run Application Locally
+
+> [!NOTE]
+><b>The application files can be downloaded as a package and then run in your favourite IDE.
+
+
+### In this article:
+* <a href="#Run-the-app-locally">Run the app locally</a>
+
+### Run the app locally
+To finally develop the app, you don't need to copy paste code. Instead use the generate app button in the toolbar. This quickly creates a package with your Angular application that can be downloaded. To run this app, open up the folder in an IDE like visual studio code. 
```

Comment:
The first sentence sounds strange. Try to rephrase it differently e.g.
To complete the app development...
Also, I think you need to add "and" between copy and paste :)
```suggestion
To finally develop the app, you don't need to copy and paste code. Instead, use the generate app button in the toolbar. This quickly creates a package with your Angular application that can be downloaded. To run this app, open up the folder in an IDE like Visual Studio Code. 
```